fetch.bs

```diff
@@ -1870,6 +1881,17 @@ is a <a>filtered response</a> whose
  <li><p>Return <var>newResponse</var>.
 </ol>
 
+<p>A <dfn id=concept-fresh-response>fresh response</dfn> is a <a for=/>response</a> whose age is
```

Comment:
Should we link "age"?

Reply:
I've adjusted it to current age and lined it.